master/static/migrations/20240617104613_handle-single-run-experiments-best-trial.tx.up.sql

```diff
@@ -0,0 +1,58 @@
+CREATE OR REPLACE FUNCTION autoupdate_exp_best_trial_single_run_no_validation() RETURNS trigger AS $$
+BEGIN 
+    WITH rc AS (
+        SELECT COUNT(*) AS count
+        FROM runs
+        WHERE experiment_id = NEW.id),
```

Comment:
Is this just trying to see if the experiment is single-trial? If so then we just check `config->searcher->name` in the experiments table, then we do not need this subquery.

Reply:
Yep! I realized this was using the old approach after looking back over this PR. I'm also swapping this over based on Nick's suggestion to be done during trial creation

Reply:
Thanks!